config/initializers/004-message_bus.rb

```diff
@@ -119,8 +119,16 @@ def setup_message_bus_env(env)
 end
 MessageBus.reliable_pub_sub.max_backlog_size = GlobalSetting.message_bus_max_backlog_size
 
-MessageBus.long_polling_enabled = SiteSetting.enable_long_polling
-MessageBus.long_polling_interval = SiteSetting.long_polling_interval
+if SiteSetting.table_exists? && SiteSetting.where(name: ['enable_long_polling', 'long_polling_interval']).exists?
+  Discourse.deprecate("enable_long_polling/long_polling_interval have switched from site settings to global settings", drop_from: '2.9.0')
```

Comment:
Can we give some suggested action here? What should people do when they see this message? Maybe something like
```suggestion
  Discourse.deprecate("enable_long_polling/long_polling_interval have switched from site settings to global settings. Remove the override from the Site Settings UI, and use environment variables to set the global settings.", drop_from: '2.9.0')
```